### DISCUSSION TITLE
Download only option

### DISCUSSION THREAD

**Discussion by ghost** [jamfprofessionalservices/JamfMigrator#113] [Ideas]
An option to not migrate but download locally could be helpful.

i would see value in having somethings backed up "offline"

**Comment by BIG-RAT** [comment]
Yes very useful. Let me know if this doesn't do what you're looking for.

[export](https://github.com/jamf/Replicator/blob/6590f9f9ff39c2830ee0af189ef92280a069909d/jamf-migrator/images/exportPrefs.png)
